pkg/api/plugin.go

```diff
@@ -77,7 +77,9 @@ type DeployFn func(context.Context, map[string]interface{}) error
 
 // PluginConfig is passed into NewPlugin
 type PluginConfig struct {
-	TestConfig TestConfig
+	SSHSrcAddrPrefix string
```

Comment:
unfortunately this is the wrong place for this - we've been working on getting everything out of PluginConfig.  It needs to go somewhere in pkg/api/admin/api/config.go.  That way it can be part of the on-disk plugin configuration in the RP.  You'll also need to update pluginconfig/pluginconfig-311.yaml - our version should be configured with '*' + a comment that MSFT would want to change that in prod

Reply:
a follow-up PR that removes PluginConfig entirely in favour of TestConfig would probably be a good plan and make things clearer for others if you'd like to do that?

Reply:
Comments help with temporary situations. In this case I will make sure this gets removed after that PR.